ubuntu_bridge_network_configuration.sh

```diff
@@ -36,18 +36,18 @@ if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
  fi
 
  if [ "$MANAGE_INT_BRIDGE" == "y" ]; then
-     # Create the baremetal bridge
-     if ! [[  $(ip a show baremetal) ]]; then
-       sudo brctl addbr baremetal
-       # sudo ifconfig baremetal 192.168.111.1 netmask 255.255.255.0 up
+     # Create the external bridge
+     if ! [[  $(ip a show external) ]]; then
```

Comment:
What is this supposed to be doing? It produces output but has no test.

Probably going for `if ! ip a show external &>/dev/null; then`  as a way to check if `external` exists?

Reply:
yeah I think the original purpose of this condition was to check whether the external interface is present or not.